code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -74,6 +75,22 @@ var/global/list/obj/item/device/pda/PDAs = list()
 		if (pen)
 			to_chat(user, "There is \a [pen] in the pen slot.")
 
+/obj/item/device/pda/New(var/mob/living/carbon/human/H)
```

Comment:
this should be on initialize

Reply:
Yoch, I might be missing something to make it work for Initialize() besides adding an argument for the ../human